sag_et.par

```diff
@@ -4,195 +4,241 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/||\
 n 7 4
+/||\
+n 9 5
+|\
 n 10 6
-/|)  /N\
+(|\  /||\
 d 12 7
 s 2 3 4 6
-/N\
+/||\

```

Comment:
It seemed like Scala supported either || ||| !! !!! style or N M U W style. I switched over to the || ||| !! !!! style. I had never seen these N M U W characters anywhere else, including while using Scala.